tvos-types.d.ts

```diff
@@ -31,31 +31,32 @@ declare module 'react-native' {
   nextFocusUp?: number,
   }
 
-  export const useTVEventHandler: (handleEvent: (event: HWKeyEvent) => void) => void;
+  export const useTVEventHandler: (handleEvent: (event: HWEvent) => void) => void;
 
-  export const TVMenuControl: {
+  export const TVEventControl: {
     enableTVMenuKey(): void;
     disableTVMenuKey(): void;
+    enableTVPanGesture(): void;
+    disableTVPanGesture(): void;
   };
 
-  interface HWFocusEvent {
-    eventType: 'blur' | 'focus';
-    eventKeyAction: -1;
-    tag: number;
-  }
-
-  export type HWKeyEvent =
-    | HWFocusEvent
-    | {
-        eventType: 'up' | 'down' | 'right' | 'left' | string;
-        eventKeyAction: -1 | 1 | 0 | number;
-        tag?: number;
-      };
+  export type HWEvent = {
+    eventType: 'up' | 'down' | 'right' | 'left' | 'blur' | 'focus' | 'pan' | string;
+    eventKeyAction: -1 | 1 | 0 | number;
```

Comment:
eventKeyAction should be optional (Apple TV doesn't send this)

Reply:
Fixed! 

I also noticed that I had the wrong casing on the body state